README.md

```diff
@@ -113,6 +113,12 @@ Not available at the moment.
 - Twitter: [@twitterhandle](https://twitter.com/the-wayword1)
```

Comment:
| @Cisco-the-wayword Twitter Link |
|-|
| ![image](https://github.com/Cisco-the-wayword/BlogApp/assets/93039359/07c9638d-9252-4eeb-9ba3-cbcc4173f7a2) |

- [ ] It would be nice if you fix your Twitter link because it is broken 😨 Kindly fix it to make your readme descriptive and professional 😊